src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/edit.html.twig

```diff
@@ -68,6 +68,16 @@
         </div>
       {% endblock %}
 
+      {% block product_tab_combinations %}
```

Comment:
That's one thing that would be great if we can improve this The idea is that I'd like this whole block to be **outside** of the product form So that no input from the combination list can mess with the product form, and even when Enter key is pressed it doesn't submit it (since it's outside..)

I think there might be something to cheat with the navigation bar, even if it implies performing the switch manually instead of relying on bootstrap nav bar

But I guess it can be done in another PR, especially because I have a small component in a pending PR that handles the automatic tab switching on page load, so there's something to do with it I think

So don't think about this for now, but maybe if you could just remove the safety for Enter key for now so that we don't forget to remove it later?